frontend/libs/studio-components/src/components/StudioModal/StudioModal.module.css

```
@@ -7,6 +7,7 @@
   --modal-translate: calc(var(--modal-position) * -1);
   --heading-height: 60px;
   --modal-border-size: 1px;
+  --modal-border-radius: 20px;
```

Denne verdien kan vi godt hente fra designsystemet. Designsystemets `Modal` bruker `--fds-border_radius-medium`, så jeg foreslår at vi bruker denne her også:
```suggestion
  --modal-border-radius: var(--fds-border_radius-medium);
```
På sikt bør vi uansett erstatte `ReactModal` med designsystemets komponent.